### PR TITLE
Fix e2e tests

### PR DESCRIPTION
**Pull request by imgerson**
This PR updates `@wordpress/env` and allows the e2e tests to run in different PHP versions. This is going to pave the way to adding support for PHP 8.

### CODE REVIEW

**Review by bfeddern-godaddy — APPROVED** [review]
Looks good to me, but might be wise to wait for one of the others to approve before merging.  I'll follow-up with you later on where/how you tested this locally or in my 3la (just so I can learn how).

**Review by jacknm — APPROVED** [review]

**Review by pantuza — APPROVED** [review]

**Comment by imgerson** [comment]
Thanks for the review @bfeddern-godaddy @jacknm @pantuza 🖖!!!

**Pull request merged**
